spiceypy/spiceypy.py

```diff
@@ -74,6 +74,33 @@
 _SPICE_EK_MAXQSEL = 100  # Twice the 50 in gcc-linux-64
 _SPICE_EK_EKRCEX_ROOM_DEFAULT = 100  # Enough?
 
+import threading
+
+spicelock = threading.Lock()
```

Comment:
I am new in the multithread field and I used your comment to better understand the documentation. Thanks.
The Rlock is better in this case because it handles recursive calls by the current thread. I will substitute that.